### PR TITLE
Update rollouts

### PR DESCRIPTION
**Pull request by IllagerCaptain**

### CODE REVIEW

File: experiment-rollout/data.json [hunk 1]
```diff
@@ -189,12 +177,33 @@
...
+                ],
+                "rate": 100,
+                "ranges": [
+                    [
+                        0,
+                        10000
+                    ]
+                ]
+            }
+        ]
```
Comment (Tolga1452) [inline experiment-rollout/data.json]:
```suggestion
        "rate": 0
```
it's lockdown mode, not member safety, and seems this data is old

**Review by Tolga1452 — CHANGES_REQUESTED** [review]

File: experiment-rollout/data.json (hunk 1 above)
Comment (IllagerCaptain) [reply experiment-rollout/data.json]: That's [what the API reports](https://rollouts.advaith.io/#2023-05_guild_member_safety_experiment), but it seems my servers that *should* have it doesn't, so I'll remove that.

Comment (IllagerCaptain) [reply experiment-rollout/data.json]: > it's lockdown mode, not member safety, and seems this data is old

That's expressing that it's rolled out to guilds with the feature, I set the default rate wrong. I have seen this feature not too uncommonly.
See https://rollouts.advaith.io/#2023-04_guild_alert_mode.

I've removed this anyways though.

**Review by Tolga1452 — APPROVED** [review]

File: experiment-rollout/data.json (hunk 1 above)
Comment (Tolga1452) [reply experiment-rollout/data.json]: oh sorry i saw now

Comment (Tolga1452) [reply experiment-rollout/data.json]: but i don't think it's usable right now, so this is better for now

**Pull request merged**